crates/egui/src/style.rs

```diff
@@ -2062,6 +2065,7 @@ impl Visuals {
             image_loading_spinners,
 
             numeric_color_space,
+            disabled_alpha: fade_out_by,
```

Comment:
Let's use the same name everywhere

```suggestion
            disabled_alpha,
```